docs/dev/policy_documents/merging-and-promoting-conventions.md

```diff
@@ -0,0 +1,40 @@
+# Merging and Promoting Conventions
+
+## Summary
+[summary]: #summary
+
+This policy defines a 'fast-track' merge and promote policy for a subset of core-plans.  
+
+## Motivation
+[motivation]: #motivation
+
+If someone who contributes heavily to core plans, has a pull request for a minor change open for 7days +, with all tests passing, it would be good if the same person can also merge and promote if there is a lack of peer reviews. One contributor (Graham Wilson, predominant) has hundreds of updates in his backlog, but has been holding back because of delays in the review process.
```

Comment:
```suggestion
Our code review process is currently slow, frustrating core-plan maintainers who are donating their personal time or their employer's time to the project. One contributor (Graham Wilson, predominant) has hundreds of updates in his backlog, but has been holding back because of delays in the review process.

If someone who contributes heavily to core plans has a pull request (PR) for a minor change open for 7+ days, the automated testing already in place and the small scope of the change makes the limited value of additional code-review small compared to the cost it is imposing on some maintainers.  In this situation, allowing veteran maintainers to use their own judgement and immediately merge the PR will help ensure core-plan maintainers can make the most of the time they spend on this project.
```

Reply:
Perfect!